### PR TITLE
Added column header change content under rows and columns.

### PR DESCRIPTION
**Pull request by Vasanth-3519**
**Description:**
Added how to change the header text in column under rows and columns with sample code.

### CODE REVIEW

File: ej2-javascript/spreadsheet/rows-and-columns.md [hunk 1]
```diff
@@ -273,6 +273,39 @@ The following code example shows how to change the width for single/multiple col
 {% previewsample "page.domainurl/code-snippet/spreadsheet/column-width-cs1" %}
 {% endif %}
 
+## Modify the column header's text
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/rows-and-columns.md:276]: Use "Change the text in the column headers"

File: ej2-javascript/spreadsheet/rows-and-columns.md [hunk 2]
```diff
@@ -273,6 +273,39 @@ The following code example shows how to change the width for single/multiple col
...
+
+You can change the column header's text using the `beforeCellRender` event. And check whether the rendered cell is `headerCell` using `e-header-cell` class and modify its `innerHTML` based on the user-defined text.
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/rows-and-columns.md:278]: Use the below content and also map the API link to the event.

Using the `beforeCellRender` event, you can change the text in the column headers. In that event, you can use the `e-header-cell` class to identify the header cell element and update its text value.

File: ej2-javascript/spreadsheet/rows-and-columns.md [hunk 3]
```diff
@@ -273,6 +273,39 @@ The following code example shows how to change the width for single/multiple col
...
+
+The following code example shows how to modify the column header's text in the spreadsheet.
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/rows-and-columns.md:280]: Use the below content

The following code example shows how to change the text in the column headers.

File: ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/system.config.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        compilerOptions: {
+            target: "umd",
+            module: "commonjs",
+            moduleResolution: "node",
+            emitDecoratorMetadata: true,
+            experimentalDecorators: true
+        }
+    },
+    paths: {
+        "syncfusion:": "https://cdn.syncfusion.com/ej2/20.3.56/"
```
Comment (gobalakrishnans) [inline ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/system.config.js:13]: Use latest CDN link

File: ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/ts/index.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="description" content="Typescript UI Controls" />
+        <meta name="author" content="Syncfusion" />
+        <link rel="shortcut icon" href="resources/favicon.ico" />
+        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
+        <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-base/styles/material.css" rel="stylesheet" />
+        <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-inputs/styles/material.css" rel="stylesheet" />
+        <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-buttons/styles/material.css" rel="stylesheet" />
+        <link href="https://cdn.syncfusion.com/ej2/20.3.56/ej2-splitbuttons/styles/material.css" rel="stylesheet" />
```
Comment (gobalakrishnans) [inline ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/ts/index.html:15]: Use latest CDN link to all CSS

File: ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/index.ts [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+let spreadsheet: Spreadsheet = new Spreadsheet({
+    beforeCellRender: (args: CellRenderEventArgs) => {
+      if (
+        args.colIndex >= 0 &&
+        args.colIndex <= 10 &&
+        args.element.classList.contains('e-header-cell')
+      ) {
+        let text: string = 'custom header ' + args.colIndex.toString();
+        args.element.innerHTML = text;
```
Comment (gobalakrishnans) [inline ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/index.ts:14]: Don't use the innerHTML , use innerText property or text

**Review by gobalakrishnans — CHANGES_REQUESTED** [review]

File: ej2-javascript/spreadsheet/rows-and-columns.md (hunk 1 above)
Comment (babu-periyasamy) [reply ej2-javascript/spreadsheet/rows-and-columns.md:276]: Changed the content.

File: ej2-javascript/spreadsheet/rows-and-columns.md (hunk 2 above)
Comment (babu-periyasamy) [reply ej2-javascript/spreadsheet/rows-and-columns.md:278]: Changed the content.

File: ej2-javascript/spreadsheet/rows-and-columns.md (hunk 3 above)
Comment (babu-periyasamy) [reply ej2-javascript/spreadsheet/rows-and-columns.md:280]: Changed the content.

File: ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/index.ts (hunk 6 above)
Comment (babu-periyasamy) [reply ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/index.ts:14]: Checked and changed to innerText.

File: ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/system.config.js (hunk 4 above)
Comment (babu-periyasamy) [reply ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/system.config.js:13]: Added the latest CDN link.

File: ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/ts/index.html (hunk 5 above)
Comment (babu-periyasamy) [reply ej2-javascript/code-snippet/spreadsheet/column-header-change-cs1/ts/index.html:15]: Added the latest CDN links.

File: ej2-javascript/spreadsheet/rows-and-columns.md [hunk 7]
```diff
@@ -273,6 +273,39 @@ The following code example shows how to change the width for single/multiple col
 {% previewsample "page.domainurl/code-snippet/spreadsheet/column-width-cs1" %}
 {% endif %}
 
+## Change the text in the column headers
```
Comment (harivenkateshe) [inline ej2-javascript/spreadsheet/rows-and-columns.md:276]: Change title like - "Changing text in column headers" for the below topic.

**Review by gobalakrishnans — APPROVED** [review]

**Pull request merged**